prime-router/settings/organizations.yml

```diff
@@ -1348,6 +1348,207 @@
   jurisdiction: STATE
   stateCode: PA
   receivers:
+    - name: elr-chester-hl7
+      organizationName: pa-phd
+      topic: covid-19
+      customerStatus: active
+      jurisdictionalFilter: [ "matches(ordering_facility_state, PA)", "matches(ordering_facility_county, Chester, Delaware)" ]
+      translation:
+        type: HL7
+        useBatchHeaders: true
+        nameFormat: chester_hl7
+        receivingApplicationName: PA-ELR
+        receivingFacilityName: PADOH
+        suppressHl7Fields: PID-13-1,ORC-23-1,OBR-16
+        replaceValue:
+          ORC-21-2: L
+        valueSetOverrides:
+          hl70189:
+            name: hl70189
+            system: HL7
+            version: 2.9
+            reference: HL7 guidance for ethnicity, modified for PA standards
+            values:
+              - code: H
+                display: Hispanic or Latino
+              - code: NH
+                replaces: N
+                display: Non Hispanic or Latino
+              - code: U
+                display: Unknown
+      transport:
+        type: SOAP
+        endpoint: "http://prime-router-soap-webservice-1:8080/castlemock/mock/soap/project/eUAvaN/BasicHttpBinding_IUploadFile"
+        soapAction: "http://nedss.state.pa.us/2012/B01/elrwcf/IUploadFile/UploadFiles"
+        credentialName: DEFAULT-SOAP
+        namespaces:
+          "xmlns:elr": "http://nedss.state.pa.us/2012/B01/elrwcf"
```

Comment:
A SOAP transport type takes the endpoint we are connecting to, and the SOAP action we will invoke. It also accepts a credential name like SFTP transports, and then it was a collection of namespaces. The namespaces reflect the namespaces used to create the XML document that gets wrapped in the SOAP envelope. I'm not super crazy about storing it here, but I couldn't talk myself into it being better housed somewhere else. So it stays here for now.